report/sql_tasks/tasks/report/create_from_scratch/99_hubspot_write/01_company_property_update/02_hubspot_export.py

```diff
@@ -0,0 +1,46 @@
+import csv
```

Comment:
I reckon leaving this out of this PR is easier that trying to insert the numbers in other keys etc.

We can deploy the view, check the number in metabase and then in a separate PR send them to HS. I guess we could also check the numbers just with the query in metabaes but creating the view feels better.